package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@remoteoss/json-schema-form",
-  "version": "0.6.5-beta.0",
+  "version": "0.6.6-dev.20231026142809",
```

Comment:
@davelange, new version released: `0.6.6-dev.20231026142809`!

One thing, pls make sure to update your branch with master because we are already on 0.7.x

Reply:
@davelange could you revert the dev version? We have a script for that, it's `version_as_main`.

Reply:
Done 👍